app/Cargo.toml

```diff
@@ -40,3 +40,6 @@ icon = "alph_32x32.gif"
 
 [package.metadata.ledger.flex]
 icon = "alph_40x40.gif"
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(target_os, values("stax", "flex", "nanox", "nanosplus"))'] }
```

Comment:
The official app demo has also introduced a similar config: https://github.com/LedgerHQ/app-starknet/blob/7ac60b8cc01b239d2053c2a552b44c876793bc9e/starknet/Cargo.toml#L38